products/BellHybrid/apps/application-bell-alarm/presenter/BellAlarmSetPresenter.hpp

```diff
@@ -14,6 +15,14 @@ namespace app
 
 namespace app::bell_alarm
 {
+    enum class Layout
+    {
+        undefined,
```

Comment:
Please capitalize also the first letter so that all enum fields are in PascalCase convention